packages/framework-provider-aws-infrastructure/src/infrastructure/params.ts

```diff
@@ -18,7 +18,7 @@ export function lambda(
   apis: APIs
 ): Pick<FunctionProps, 'runtime' | 'timeout' | 'memorySize' | 'environment'> {
   return {
-    runtime: Runtime.NODEJS_14_X,
+    runtime: Runtime.NODEJS_18_X,
```

Comment:
Azure infrastructure also defines the runtime version in **terraform-function-app.ts** file. Current version is Node 14:

```
      siteConfig: {
        applicationStack: {
          nodeVersion: '~14',
        },
      },
```

I think we should update it too

Reply:
Also I think the `functionsExtensionVersion: '~3'` needs to be updated to `functionsExtensionVersion: '~4'` in the same file